lib/sorcery/controller/submodules/brute_force_protection.rb

```diff
@@ -20,7 +20,7 @@ module InstanceMethods
 
           # Increments the failed logins counter on every failed login.
           # Runs as a hook after a failed login.
-          def update_failed_logins_count!(credentials)
+          def update_failed_logins_count!(credentials, _failure_reason)
```

Comment:
This appears to be the only callback that was changed, and failure_reason is unused. @pinewong can you elaborate on this change?

Reply:
`failure_reason` is useful for `after_failed_login` callback, so I added it. Then I searched the current project, and found there was one place(we are referring to) use it, I have to modify it synchronously.

Reply:
Ah, yeah that makes sense.